code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -25,6 +25,31 @@
 	var/trashtype = null //For disposable cuffs
 	var/ignoresClumsy = FALSE
 
+/obj/item/restraints/handcuffs/Initialize(mapload)
+	. = ..()
+	// yes, this is an empty icon. Make it not visible in hand.
+	AddComponent(/datum/component/two_handed, \
+		wield_callback = CALLBACK(src, PROC_REF(wield)), \
+		unwield_callback = CALLBACK(src, PROC_REF(unwield)) \
+	)
+
+/obj/item/restraints/handcuffs/proc/wield(obj/item/source, mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	user.remove_overlay(HANDCUFF_LAYER)
+	user.overlays_standing[HANDCUFF_LAYER] = mutable_appearance('icons/mob/mob.dmi', cuffed_state, layer = -HANDCUFF_LAYER, color = color)
+	user.apply_overlay(HANDCUFF_LAYER)
+	item_state = "weijf2349-h238h9f23"  // thanks for the string contra
```

Comment:
What on earth

Reply:
Setting the item state to a random string gives us an empty one, vs. just setting it to null which I think pulls the icon state name

Reply:
Surely setting the opacity to 101 or similar is a better solution?

Reply:
Can you do that for item states? It's specifically for the in-hand overlay